src/components/Community.js

```diff
@@ -157,42 +157,34 @@ export const Community = () => {
                 <g id="Group">
                   <path
                     d="M338.114754,1017.45902 C338.114754,1110.57377 262.04918,1186.63934 168.934426,1186.63934 C75.8196721,1186.63934 -0.245901639,1110.57377 -0.245901639,1017.45902 C-0.245901639,924.344262 75.8196721,848.278689 168.934426,848.278689 L338.114754,848.278689 L338.114754,1017.45902 L338.114754,1017.45902 Z"
-                    id="Shape"
```

Comment:
The `"id"` attribute is meant to uniquely identify elements on a page. Having multiple of the same, even within an `svg`, can confuse tools such as screenreaders that try to use that uniqueness.

In this case the `id` looks like an artifact of whatever tool generated this image so... yay! Slight dead code deletion!

https://dequeuniversity.com/rules/axe/3.5/duplicate-id